content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -25,7 +25,13 @@ it succeeds, making sure those probes don't interfere with the application start
 This can be used to adopt liveness checks on slow starting containers, avoiding them
 getting killed by the kubelet before they are up and running.
 
+{{< caution >}}
+Liveness probes can be a powerful way to recover from application failures, but they should be used with caution. Liveness probes must be configured carefully to ensure that they truly indicate unrecoverable application failure, for example a deadlock. A common pattern for liveness probes is to use the same low-cost HTTP endpoint as for readiness probes, but with a higher failureThreshold. This ensures that the pod is observed as not-ready for some period of time before it is hard killed.
```

Comment:
Please wrap the long lines manually as we do for all other paragraphs.

The statement sounds like the Liveness probe is used to detect readiness, which is incorrect.

Reply:
In my opinion, I like the first part which explains the benefits of liveness probes in a positive way, but also emphasizes the need for caution in their implementation.

Perhaps there is a way to avoid repetition with the first line of the liveness documentation.

As for the "common pattern" section, I'm not sure if it needs to be mentioned at this point.

In my opinion, the decision to use probes should depend on how the application is developed.

I think it's an overused pattern, and it's important to have more discussion about its impact and how to avoid misdirection.

Reply:
@tengqm & @binboum May you both please review this:

 **_"Liveness probes can be a powerful way to recover from application failures, but they should be used with caution. It indicates unrecoverable application failure, for example a deadlock, which is why it must be configured carefully. Liveness probe uses low-cost HTTP endpoint with a higher failureThreshold."_**